crates/orchestrator/src/queue/job_queue.rs

```diff
@@ -20,9 +21,23 @@ use crate::workers::snos::SnosWorker;
 use crate::workers::update_state::UpdateStateWorker;
 use crate::workers::Worker;
 
-pub const JOB_PROCESSING_QUEUE: &str = "madara_orchestrator_job_processing_queue";
-pub const JOB_VERIFICATION_QUEUE: &str = "madara_orchestrator_job_verification_queue";
-// Below is the Data Letter Queue for the above two jobs.
+pub const SNOS_JOB_PROCESSING_QUEUE: &str = "madara_orchestrator_snos_job_processing_queue";
+pub const SNOS_JOB_VERIFICATION_QUEUE: &str = "madara_orchestrator_snos_job_verification_queue";
+
+pub const PROVING_JOB_PROCESSING_QUEUE: &str = "madara_orchestrator_proving_job_processing_queue";
+pub const PROVING_JOB_VERIFICATION_QUEUE: &str = "madara_orchestrator_proving_job_verification_queue";
+
+pub const PROOF_REGISTRATION_JOB_PROCESSING_QUEUE: &str = "madara_orchestrator_proof_registration_job_processing_queue";
+pub const PROOF_REGISTRATION_JOB_VERIFICATION_QUEUE: &str =
+    "madara_orchestrator_proof_registration_job_verification_queue";
+
+pub const DATA_SUBMISSION_JOB_PROCESSING_QUEUE: &str = "madara_orchestrator_data_submission_job_processing_queue";
+pub const DATA_SUBMISSION_JOB_VERIFICATION_QUEUE: &str = "madara_orchestrator_data_submission_job_verification_queue";
+
+pub const UPDATE_STATE_JOB_PROCESSING_QUEUE: &str = "madara_orchestrator_update_state_job_processing_queue";
+pub const UPDATE_STATE_JOB_VERIFICATION_QUEUE: &str = "madara_orchestrator_update_state_job_verification_queue";
+
+// Below is the Dead Letter Queue for the above two jobs.
```

Comment:
nitpick: change to
Below is the single Dead Letter Queue for all the above queues.

Reply:
updated